packages/website/sidebars/sidebar.base.js

```diff
@@ -15,40 +17,64 @@ module.exports = {
         },
         'linting/configs',
         {
-          label: 'Troubleshooting & FAQs',
-          link: {
-            id: 'linting/troubleshooting',
-            type: 'doc',
-          },
           collapsible: false,
-          type: 'category',
           items: [
             'linting/troubleshooting/performance-troubleshooting',
             'linting/troubleshooting/formatting',
             'linting/troubleshooting/tslint',
           ],
+          label: 'Troubleshooting & FAQs',
+          link: {
+            id: 'linting/troubleshooting',
+            type: 'doc',
+          },
+          type: 'category',
         },
       ],
+      label: 'Getting Started',
       link: {
         id: 'getting-started',
         type: 'doc',
       },
-      label: 'Getting Started',
       type: 'category',
     },
-    'custom-rules',
     {
+      collapsible: false,
+      items: [
+        'users/dependency-versions',
+        'users/releases',
+        'users/versioning',
+      ],
+      label: 'Users',
```

Comment:
I think this is the only part of this PR I'm not 100% positive on... Everybody is a user. Is there a more precise term we can use? _End Users_ perhaps?

Reply:
My thinking was there are "users" which are the ones doing linting and the "developers" which are the ones building things with their tooling.

Technically the "developers" are "users" of our tooling, for sure, but I thought there might be enough distinction?

I don't love "end users"... I wonder if there's another term we can use?